airbyte-integrations/connectors/source-amazon-mws/source_amazon_mws/schemas/GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL.json

```diff
@@ -0,0 +1,158 @@
+{
+  "name": "GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL",
+  "supported_sync_modes": ["full_refresh", "incremental"],
+  "source_defined_cursor": true,
+  "default_cursor_field": ["purchase-date"],
+  "json_schema": {
+    "type": "object",
+    "title": "GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL",
```

Comment:
should this just be called `orders`?

Reply:
https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reportType_string_array_values.md we need to support more report Types so thought is goods to name them same as the report type